CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+<!--
+Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Changelog
+
+## 23.06 (2023-06-29)
+
+- Implemented periodic checkpointing
+- Added support for custom docker args
+- Detect and handle invalid metrics url
+- Profile will now automatically create the default detailed reports
+
+## 23.05 (2023-06-01)
+
+- `request-rate-range` can now be searched in brute mode
+- Capture PA errors in a log file
+- Added detection for Triton Server launch failures
+- Added `cpu_only` option for ensemble composing models
+- Added binary concurrency search to quick search mode
+- Added binary parameter search to brute search mode
+
+## 23.04 (2023-04-27)
+
+- Support for BLS models added
```

Comment:
does this support links? Can we link docs for new features?

Reply:
It does, and I've added them to the relevant bullet points